prober/derp.go

```diff
@@ -317,6 +317,7 @@ func (d *derpProber) probeBandwidth(from, to string, size int64) ProbeClass {
 			return []prometheus.Metric{
 				prometheus.MustNewConstMetric(prometheus.NewDesc("derp_bw_probe_size_bytes", "Payload size of the bandwidth prober", nil, l), prometheus.GaugeValue, float64(size)),
 				prometheus.MustNewConstMetric(prometheus.NewDesc("derp_bw_transfer_time_seconds_total", "Time it took to transfer data", nil, l), prometheus.CounterValue, transferTimeSeconds.Value()),
+				prometheus.MustNewConstMetric(prometheus.NewDesc("derp_bw_bytes_total", "Amount of data transferred", nil, l), prometheus.CounterValue, float64(size)),
```

Comment:
this is the same as the metric on L318 besides metric type?

Reply:
Correct. The idea is to keep a cumulative counter of bytes transferred so that we can look at the rate of bytes transferred / transfer time.

Reply:
`size` is not cumulative, it's a fixed value